apps/frontend/src/pages/root/components/user-wheel/libs/components/scores-edit-modal/scores-edit-modal.tsx

```diff
@@ -45,6 +50,10 @@ const ScoresEditModal: React.FC<Properties> = ({
 		onSaveChanges();
 	}, [onSaveChanges, dispatch, scores, isDiscardButtonDisabled]);
 
+	const areChangesScores = useCallback(() => {
+		return JSON.stringify(originalScores) !== JSON.stringify(scores);
+	}, [originalScores, scores]);
```

Comment:
I don't like this approach, we shouldn't check JSON.stringify

Reply:
I replace this logic with for of